.travis.yml

```diff
@@ -61,6 +62,22 @@ before_install:
   # process json output from github api
   - sudo apt-get install -y jq
 
+  # Install libneurosim
```

Comment:
Does this mean that libneurosim is always installed? I don't see a conditional on xLIBNEUROSIM

Reply:
As this is in accordance with the handling of all other dependencies, I would not want to change this here and rather do this in a refactoring of the whole dependency handling. A good opportunity is probably when we move to a Docker-based setup.